vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl

```diff
@@ -20711,15 +20711,6 @@ quantitykind:RankineTemperature
   rdfs:label "Rankine temperature" ;
   rdfs:label "Rankine temperature"@en-US ;
 .
-quantitykind:RateOfChangeOfTemperature
```

Comment:
I am personally ok with just removing this, but don't we have the policy to first deprecate, later remove?